src/Microsoft.Diagnostics.Runtime/src/Common/ContextHelper.cs

```diff
@@ -53,14 +54,33 @@ private static void Init()
             if (_context != null)
                 return;
 
+#if !NET45
```

Comment:
I need to go back and revisit how we determine the sizes here.  It's a much bigger change than this, so we'll check this in as-is.